src/config/CMakeLists.txt

```diff
@@ -1,8 +1,18 @@
-project(CemuConfig)
-
-file(GLOB CPP_FILES *.cpp)
-file(GLOB H_FILES *.h)
-add_library(CemuConfig ${CPP_FILES} ${H_FILES})
+add_library(CemuConfig
+    ActiveSettings.cpp
+    ActiveSettings.h
+    CemuConfig.cpp
+    CemuConfig.h
+    CMakeLists.txt
```

Comment:
Copy & paste mistake?

Reply:
Yeah-- I used `tree --noreport -i -f` but that kept directories without files and CMakeLists.txt. Fixed.